bootstrap-salt.sh

```diff
@@ -3415,9 +3415,22 @@ install_debian_check_services() {
 #
 
 install_fedora_deps() {
+    if [ -n "$_PY_EXE" ] && [ "$_PY_MAJOR_VERSION" -eq 3 ]; then
+        # Packages are named python3-<whatever>
+        PY_PKG_VER=3
+        __PACKAGES="python3-m2crypto python3-PyYAML"
+    else
+        PY_PKG_VER=2
+        __PACKAGES="m2crypto"
+        if [ "$DISTRO_MAJOR_VERSION" -ge 28 ]; then
+          __PACKAGES="${__PACKAGES} python2-pyyaml"
+        else
+          __PACKAGES="${__PACKAGES} PyYAML"
+       fi
+    fi
 
-    __PACKAGES="dnf-utils libyaml m2crypto PyYAML python-crypto python-jinja2"
-    __PACKAGES="${__PACKAGES} python2-msgpack python2-requests python-zmq"
+    __PACKAGES="dnf-utils ${__PACKAGES} python${PY_PKG_VER}-crypto python${PY_PKG_VER}-requests python${PY_PKG_VER}-zmq"
```

Comment:
I know this is picky, but can you keep the order of package names? They're in alphabetical order to help keep track of when new deps are added. I know it's a small thing, but this script is massive, so it helps with future updates.

Reply:
not at all, no problem :)